test/controllers/relation_controller_test.rb

```diff
@@ -337,6 +337,7 @@ def test_create
   def test_update_relation_tags
     basic_authorization "test@example.com", "test"
     rel_id = current_relations(:multi_tag_relation).id
+    create_list(:relation_tag, 4, :relation => current_relations(:multi_tag_relation))
```

Comment:
Now arguably here we should propagate as technically by creating tags on the current object that are not on the corresponding history object we have put the database into an invalid state. It probably doesn't actually affect anything though, but then I'm not sure we even need to create these tags? We'll still test whether adding the new one works...